src/wallet/rpcwallet.cpp

```diff
@@ -2271,8 +2271,7 @@ UniValue z_listunspent(const UniValue& params, bool fHelp)
             "    \"type\" : \"sprout|sapling|orchard\", (string) The shielded pool\n"
             "    \"jsindex\" (sprout) : n,       (numeric) the joinsplit index\n"
             "    \"jsoutindex\" (sprout) : n,       (numeric) the output index of the joinsplit\n"
-            "    \"outindex\" (sapling) : n,       (numeric) the output index\n"
-            "    \"actionindex\" (orchard) : n,    (numeric) the output index\n"
+            "    \"outindex\" (transparent, sapling, orchard) : n,       (numeric) the output index\n"
```

Comment:
```suggestion
            "    \"outindex\" (sapling, orchard) : n,  (numeric) the output index\n"
```
Also make the indentation of descriptions consistent.

Reply:
Opened #5535 for this and similar cleanups.

Reply:
After 5.0.0 one of the things I'd like to get in is https://github.com/zcash/zcash/pull/5237